vars/containerPushToRegistry.groovy

```diff
@@ -127,6 +132,10 @@ void call(Map parameters = [:]) {
                         def latestImage = "${config.dockerImage.split(':')[0]}:latest"
                         dockerUtils.moveImage([image: config.sourceImage, registryUrl: config.sourceRegistryUrl], [image: latestImage, registryUrl: config.dockerRegistryUrl, credentialsId: config.dockerCredentialsId])
                     }
+                    if (config.tagLatest) {
```

Comment:
```suggestion
                    if (config. tagArtifactVersion) {
```

Reply:
I guess you meant to check here if the `tagArtifactVersion` flag is set

Reply:
Yes, thank you.